scripts/uninstall/UnRegisterTaskQueueServices.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017-2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoAdvancedSearch\scripts\uninstall;
+
+use oat\oatbox\extension\InstallAction;
+use oat\oatbox\reporting\Report;
+use oat\tao\model\taskQueue\Queue;
+use oat\tao\model\taskQueue\QueueDispatcher;
+use oat\tao\model\taskQueue\QueueDispatcherInterface;
+use oat\taoAdvancedSearch\scripts\install\RegisterTaskQueueServices;
+
+class UnRegisterTaskQueueServices extends InstallAction
```

Comment:
I think this class together with `RegisterTaskQueueServices` could be a service and the `getNewAssociations` could be injected by OPTIONS, together with the queue name, so we can easily customize it.

Could be even a tao-core class for future cases like this. What do you think?